proxy/src/config.rs

```diff
@@ -299,23 +302,35 @@ impl<'a> TryFrom<&'a Strings> for Config {
         let tls_settings = match (tls_trust_anchors?,
                                   tls_end_entity_cert?,
                                   tls_private_key?,
-                                  pod_name?.as_ref())
+                                  tls_pod_identity_template?.as_ref(),
+                                  tls_controller_identity?)
```

Comment:
tioli: It looks like `tls_controller_identity?` doesn't actually have to be part of the match

Reply:
> tioli: It looks like tls_controller_identity? doesn't actually have to be part of the match

I admit it isn't super clear but note that it is done there to make sure that `tls_controller_identity` isn't an `Result::Err` no matter which match arm matches.